### DISCUSSION TITLE
Bluetooth Device as interface

### DISCUSSION THREAD

**Discussion by Dant3s** [markqvist/Reticulum#200] [Help & Questions] [answered]
Hi :wave:  

Quick question, since couln't find it in the docs, (I would assume is not posible.. but better to ask in the community). Is posibble for Ret to use an Rnode, but connected via BT and not USB?.

**Comment by markqvist** [comment, marked as answer]
Hi @Dant3s!

Yes, this is possible for all RNodes that have a bluetooth-capable chipset, so all ESP32-based ones for example.

Here is [a setup guide for Android](https://unsigned.io/guides/2023_01_19_RNode_Bluetooth_Setup_In_Sideband_On_Android.html).

For setup with `rnsd` or other programs on a computer, you can use `rnodeconf --bluetooth-on` and `rnodeconf --bluetooth-pair`, and then pair it with your computer. Then you just specify the bluetooth RFCOMM serial port in the Reticulum configuration.

**Reply by Dant3s** [reply]
Thanks @markqvist ! the use case was specific to the computer use case., since the Heltec v2 had the unreliable usb connection!

**Reply by markqvist** [reply]
Ah, cool. You should be able to get it working that way, then :)

**Comment by sp00ck** [comment]
Why not wifi?
why not using wifi as transport for reticulum?

**Reply by markqvist** [reply]
May I kindly direct our attention to [the Roadmap](https://github.com/markqvist/Reticulum/blob/master/Roadmap.md)?

Reticulum already works over common WiFi devices. Use of raw WiFi / 2.4 GHz RF via ESP32 is already on the roadmap as well.
